invokeai/app/services/sqlite.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, filename: str, table_name: str, id_field: str = "id"):
         self._conn = sqlite3.connect(
             self._filename, check_same_thread=False
         )  # TODO: figure out a better threading solution
+        self._conn.execute('pragma journal_mode=wal')
```

Comment:
Still trying to figure out a way to remove this. A really weird database lock is happening without it